teammapper-frontend/mmp/src/map/handlers/nodes.ts

```diff
@@ -1084,20 +1159,20 @@ export default class Nodes {
      * @param {boolean} direction
      */
     private moveSelectionOnBranch(direction: boolean) {
-        if ((this.getOrientation(this.selectedNode) === false && direction) ||
-            (this.getOrientation(this.selectedNode) === true && !direction)) {
+        if ((this.getNodeOrientation(this.selectedNode) === false && direction) ||
```

Comment:
just `!this.getNodeOrientation(this.selectedNode)` or `this.getNodeOrientation(this.selectedNode)` is enough or why the explicit check for true and false? do you want to avoid checking against null or something?